pages/api/enroll/child.ts

```diff
@@ -0,0 +1,97 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { ResponseUtil } from "@utils/responseUtil";
+import {
+    createParentRegistration,
+    getParentRegistration,
+} from "@database/enroll";
+import { validateParentRegistrationRecord } from "@utils/validation/parentRegistration";
+import { ParentRegistrationInput } from "models/ParentRegistration";
+
+/**
+ * handle controls the request made to the enroll/child resource.
+ * This endpoint is used when a parent is enrolling their child into a program
+ * @param req API request object
+ * @param res API response object
+ */
+export default async function handle(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    switch (req.method) {
+        case "GET": {
+            // obtain query parameters
+            const { studentId, classId } = req.query;
+
+            // verify that query parameters were passed in
+            if (!studentId || !classId) {
+                return ResponseUtil.returnBadRequest(
+                    res,
+                    "studentId and classId are required to obtain a program registration record",
+                );
+            }
+
+            // parse query parameters from strings to numbers
+            const studentIdNumber = parseInt(studentId as string, 10);
```

Comment:
What happens if you don't specify a radix?

Reply:
According to [this ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt)- it doesn't default to base 10 so it's necessary to have it